pkg/executor/task_handle.go

```diff
@@ -77,3 +77,15 @@ func StopAndEraseOutput(handle TaskHandle) (errorCollection errcollection.ErrorC
 
 	return
 }
+
+// GetWaitChannel returns channel that will return result (any encountered error) of
+// Wait() method in provided handle.
+func GetWaitChannel(handle TaskControl) <-chan error {
```

Comment:
I'm not convinced that `waitForTask` is good name for this function. `waitForTask` suggest waiting, and waiting is *blocking*. The purpose of this function is to translate sync API to asynchronous one. And the function name exactly tells user what one will receive - a channel with Wait method.

I did it public, as it could be used as a tool for building multi-waiting experiments (e.g. wait for multiple load generators + timeout), so that user can have an option.